From 3edaa3875fbeb0b2effd77c62baabf2933efc6ef Mon Sep 17 00:00:00 2001
From: Dom Cobley <popcornmix@gmail.com>
Date: Fri, 13 Sep 2024 17:23:58 +0100
Subject: [PATCH 1301/1350] dts: 2712: Reduce default cma usage on Pi5

Significant cma shouldn't really be needed on Pi5 as the hardware
blocks support iommu and can access system memory.

We've migrated codec and camera support to system memory, and 3d
has always (even on Pi4) used system memory.

A large cma block causes issues with enabling NUMA on a low
memory (2G) Pi5, as cma cannot span numa regions.

Signed-off-by: Dom Cobley <popcornmix@gmail.com>
---
 arch/arm/boot/dts/overlays/vc4-kms-v3d-pi5-overlay.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/arch/arm/boot/dts/overlays/vc4-kms-v3d-pi5-overlay.dts
+++ b/arch/arm/boot/dts/overlays/vc4-kms-v3d-pi5-overlay.dts
@@ -3,7 +3,7 @@
 #include "cma-overlay.dts"
 
 &frag0 {
-	size = <((320-4)*1024*1024)>;
+	size = <(64*1024*1024)>;
 };
 
 / {
